article.tex

```diff
@@ -810,6 +835,21 @@ \subsubsection*{Aromaticity}
 \end{verbatim}
 \vspace{0.2cm}
 
+For SMARTS matching and SMILES generation a model based
+on Daylight \cite{DaylightCIS} can be used and offers significant
+speed improvements to the one based on CDK Atom Types. 
+This model has recently been documented as part of the
+OpenSMILES specification (\url{http://opensmiles.org/}):
+
+\vspace{0.2cm}
+\begin{verbatim}
+Aromaticity aromaticity = new Aromaticity(
+  ElectronDonation.daylight(), Cycles.all()
+);
+\end{verbatim}
+\vspace{0.2cm}
+
```

Comment:
oh, I really love to sit down with you on the resonance structures needed for Gasteiger pi charges...